shuup/gdpr/admin_module/toolbar.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2018, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext_lazy as _
+
+from shuup.admin.toolbar import PostActionButton
+
+
+class AnonymizeContactToolbarButton(PostActionButton):
+    def __init__(self, object, **kwargs):
+        kwargs["icon"] = "fa fa-user-times"
+        kwargs["text"] = _("Anonymize")
+        kwargs["confirm"] = _(
+            "This action will replace all contact personal data with random values making it "
+            "impossible to be identified. The account will also be deactivated and the pending order will be canceled. "
```

Comment:
`The account will also be deactivated and the pending order will be canceled` =>  `The account will also be deactivated and any pending order(s) will be canceled`